analyzer/src/funTest/assets/projects/synthetic/spdx/project/project-with-external-ref-with-local-ref.spdx.yml

```diff
@@ -0,0 +1,51 @@
+SPDXID: "SPDXRef-DOCUMENT"
+spdxVersion: "SPDX-2.2"
+creationInfo:
+  created: "2020-07-23T18:30:22Z"
+  creators:
+  - "Organization: Example Inc."
+  - "Person: Max Mustermann"
+  licenseListVersion: "3.9"
+name: "xyz-0.1.0"
+dataLicense: "CC0-1.0"
+documentNamespace: "http://spdx.org/spdxdocs/spdx-document-xyz"
+documentDescribes:
+- "SPDXRef-Package-xyz"
+externalDocumentRefs:
+- externalDocumentId: "DocumentRef-curl-7.70.0"
+  checksum:
+    algorithm: "SHA1"
+    checksumValue: "a2ab98f8e10bad892313ccc573b3820369551dcb"
+  spdxDocument: "file:///home/stephanie/Projects/ort_projects/oss-review-toolkit/analyzer/src/funTest/assets/projects/synthetic/spdx/package/libs/curl/package.spdx.yml"
```

Comment:
This is a file path on your system, I presume this fails when during CI. Better make this a relative path